testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/base/HttpClientTest.groovy

```diff
@@ -467,6 +519,10 @@ abstract class HttpClientTest extends AgentInstrumentationSpecification {
     true
   }
 
+  boolean testCausality() {
+    true
```

Comment:
Should we default to false here until we've got most of the instrumentation happy with these tests? Or is better to default to true and explicitly call out where we aren't working yet?

Reply:
I think the latter, yes. This way it will be easier to remedy the situation: just try to remove overrides one by one